voice/bxml/callbacks/recordComplete.md

```diff
@@ -0,0 +1,53 @@
+{% method %}
+##  Record Complete event
+
+The Record Complete event is sent after a [`<Record>`](../verbs/record.md) verb has executed and the BXML returned by this callback is executed next.
+
+At a later moment the [Recording Available](recordingAvailable.md) event is sent asynchronously once the recording is available for download.
```

Comment:
I would say "When the recording is available for download, a Recording Available event will be sent."